Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM rocker/shiny-verse:latest
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    sudo \
+    libxt-dev \
+    libssl-dev \
+    libssh2-1-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install -y \
+    sudo \
+    pandoc \
+    pandoc-citeproc \
+    libcurl4-gnutls-dev \
+    libcairo2-dev \
+    libxt-dev \
+    libssh2-1-dev
+
+RUN R -e "install.packages('shiny', repos='http://cran.rstudio.com/')"
+RUN R -e "install.packages('shinythemes', repos='http://cran.rstudio.com/')"
+RUN R -e "install.packages('plotly', repos='http://cran.rstudio.com/')"
+RUN R -e "install.packages('ggbeeswarm', repos='http://cran.rstudio.com/')"
+RUN R -e "install.packages('reshape2', repos='http://cran.rstudio.com/')"
+RUN R -e "install.packages('shinycssloaders', repos='http://cran.rstudio.com/')"
+RUN R -e "install.packages('beeswarm', repos='http://cran.rstudio.com/')"
+RUN R -e "install.packages('RCurl', repos='http://cran.rstudio.com/')"
+RUN R -e "install.packages('data.table', repos='http://cran.rstudio.com/')"
+RUN R -e "install.packages('RMariaDB', repos='http://cran.rstudio.com/')"
+
+## Install packages from CRAN
+RUN install2.r --error \
+    -r 'http://cran.rstudio.com' \
+    googleAuthR \
+    ## install Github packages
+    ## clean up
+    && rm -rf /tmp/downloaded_packages/ /tmp/*.rds## assume shiny app is in build folder /shinyCOPY ./Shiny/ /srv/shiny-server/shiny/# select portEXPOSE 3838
```

Comment:
Are these inline comments? and is it all supposed to be commented out?

Reply:
Admittedly this was copied from someone else. We don't need any of it. I just removed it. See [25133c8](https://github.com/leylabdotorg/amlviz/pull/24/commits/25133c83976da26fd34e097a89a3808f10919f3e)